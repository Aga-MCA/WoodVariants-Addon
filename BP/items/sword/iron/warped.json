{"format_version":"1.21.0","minecraft:item":{"description":{"identifier":"aga_wood_variants:iron_warped_sword","menu_category":{"category":"equipment","group":"itemGroup.name.sword"}},"components":{"minecraft:repairable":{"repair_items":[{"items":["minecraft:iron_ingot"],"repair_amount":62},{"items":["aga_wood_variants:iron_warped_sword"],"repair_amount":"math.min(q.remaining_durability+c.other->q.remaining_durability+math.floor(q.max_durability/20),c.other->q.max_durability)"}]},"minecraft:max_stack_size":1,"minecraft:hand_equipped":true,"minecraft:durability":{"max_durability":250},"minecraft:damage":{"value":6},"minecraft:enchantable":{"value":14,"slot":"sword"},"minecraft:can_destroy_in_creative":false,"minecraft:digger":{"use_efficiency":false,"destroy_speeds":[{"block":{"tags":"q.any_tag('destroy:sword','minecraft:is_sword_item_destructible','minecraft:is_tool_item_destructible','minecraft:is_digger_item_destructible')"},"speed":6},{"block":"minecraft:web","speed":6},{"block":"minecraft:bamboo","speed":6}]},"minecraft:icon":"iron_warped_sword","minecraft:tags":{"tags":["minecraft:is_tool","minecraft:is_sword","aga:durability","minecraft:iron_tier"]}}}}