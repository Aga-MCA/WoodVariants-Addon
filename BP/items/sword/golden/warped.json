{"format_version":"1.21.0","minecraft:item":{"description":{"identifier":"aga_wood_variants:golden_warped_sword","menu_category":{"category":"equipment","group":"itemGroup.name.sword"}},"components":{"minecraft:max_stack_size":1,"minecraft:hand_equipped":true,"minecraft:durability":{"max_durability":31},"minecraft:damage":{"value":4},"minecraft:enchantable":{"value":25,"slot":"sword"},"minecraft:can_destroy_in_creative":false,"minecraft:digger":{"use_efficiency":false,"destroy_speeds":[{"block":{"tags":"q.any_tag('destroy:sword','minecraft:is_sword_item_destructible','minecraft:is_tool_item_destructible','minecraft:is_digger_item_destructible')"},"speed":12},{"block":"minecraft:web","speed":12},{"block":"minecraft:bamboo","speed":12}]},"minecraft:icon":"golden_warped_sword","minecraft:tags":{"tags":["minecraft:is_tool","minecraft:is_sword","aga:durability","minecraft:golden_tier"]}}}}